packages/tx/test/typedTxsAndEIP2930.spec.ts

```diff
@@ -88,6 +88,37 @@ tape(
       t.end()
     })
 
+    t.test('cannot input decimal values', (st) => {
+      const values = ['chainId', 'nonce', 'gasPrice', 'gasLimit', 'value', 'v', 'r', 's']
+      const cases = [
+        10.1,
+        '10.1',
+        '0xaa.1',
+        -10.1,
+        '-10.1',
+        '-0xaa',
+        Infinity,
+        -Infinity,
+        NaN,
+        {},
```

Comment:
Perhaps also add `null` and `undefined`

Reply:
`undefined` won't work since that's the default field of an object if an attribute does not exist (tx will then choose a default value)

Reply:
`null` also fails tests

Reply:
Ok thanks for clarifying 🙂